doc/test-cases.md

```diff
@@ -0,0 +1,36 @@
+# Test Cases
+## Purpose
+Goal of this document is to document non trivial testing scenarious we can use to ensure network module is not seriously broken.
+
+## Test Cases
+
+### Complex setup
+
+Test case require VM with two network cards. Setup is:
+
+1. create bonding on top of two cards
+2. create bridge on top of bond
+3. create two vlans on top of bridge
+4. create tun and tap on top bridge
+5. create dummy
+6. create vlan on top of dummy
+
+#### CLI
+
+For testing this scenario create script that take as params name of two physical devices and using yast2 lan CLI create testing setup.
+
+#### AutoYaST
+
+For testing this scenario create autoyast profile that can be passed to autoinstallation that creates required setup.
```

Comment:
We should be able to reproduce the rest of scenarios but using AutoYaST profiles. Moreover, we should be able that the base case (no network configuration at all) still works.